### ISSUE TITLE
lob_epoch_trainer tiny bug

### ISSUE THREAD

**Issue by alfredyuan** [passalis/dain#1]
In line 39 of train_utils.py

It should be:

loss = (train_loss / counter ).cpu().data.numpy()

otherwise train_loss is not used and loss seems to be a pretty small number.

Can you please double check?

**Comment by breznak** [comment]
This could be closed as it seems to be correct in the current code: 
```
    loss = (loss / counter).cpu().data.numpy()
    return loss
```
Thanks for providing the solution!
